### PR TITLE
tweaks to ref-based rnaseq tutorial

### PR DESCRIPTION
**Pull request by shiltemann**
The HISAT2 wrapper has been updated since this tutorial was made, so I changed the parameter descriptions to match. I also got different results when performing the mapping so I adjusted the answers to affected questions as well.

I added `htseq_count` and `rseqc` to tools.yaml file (the latter for the "infer experiment" tool referred to in the library determination step)

ping @malloryfreeberg 

P.S. I am teaching this module on Monday, a merge before then would be much appreciated

### CODE REVIEW

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 1]
```diff
@@ -51,20 +53,13 @@ We have extracted sequences from the Sequence Read Archive (SRA) files to build
...
->    > * Press **Save**
->    {: .tip}
->
->    As default, Galaxy uses the link as the name of the new dataset. It also does not link the dataset to a database or a reference genome.
->
->    > ### {% icon comment %} Comments
->    > - Edit the "Database/Build" to select "dm3"
->    > - Rename the datasets according to the samples
->    {: .comment}
+> 3. Change the file type from `fastq` to `fastqsanger` once the data file is in your history
```
Comment (mvdbeek) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md]: This should default to fastqsanger in 17.09, maybe we could say:
```
Once the data file is in your history verify that the datatype is `fastqsanger`, not `fastq`. If the datatype is `fastq` change the file type from `fastq` to `fastqsanger` 
```

File: topics/transcriptomics/tutorials/ref-based/tutorial.md [hunk 2]
```diff
@@ -184,9 +179,9 @@ We can now try to determine the library type of our data.
 >
 > 1. Load the Ensembl gene annotation for *Drosophila melanogaster* ([`Drosophila_melanogaster.BDGP5.78.gtf`](https://zenodo.org/record/290221/files/Drosophila_melanogaster.BDGP5.78.gtf)) from [Zenodo](https://dx.doi.org/10.5281/zenodo.290221) into your current Galaxy history and rename it
 >
-> 2. **Select first** {% icon tool %}: Downsample the both FastQ files generated by Trim Galore to 200k or 1M reads
+> 2. **Select first** {% icon tool %}: Downsample both the FastQ files generated by Trim Galore to 200k or 1M reads
```
Comment (mvdbeek) [inline topics/transcriptomics/tutorials/ref-based/tutorial.md]: I think the `the` here is not necessary

**Comment by shiltemann** [comment]
thanks @mvdbeek, made the changes :)

**Comment by bebatut** [comment]
Looks good for me too!

**Pull request merged**